src/features/user/DonationReceipt/microComponents/ReceiptActions.tsx

```diff
@@ -30,9 +35,7 @@ const ReceiptActions = ({
             text={tReceipt('modifyContactInformation')}
             elementType="button"
             icon={<EditIcon />}
-            onClick={() =>
-              router.push(`/profile/tax-receipt/modify-recipient-data`)
-            }
+            onClick={redirectToUserDataForm}
```

Comment:
It would be more appropriate for this to be a link element instead of a button element.

Configure WebappButton appropriately.